src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,93 @@
+package lotto.view;
+
+import static lotto.util.ConstantUtils.*;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import lotto.util.ValidationUtils;
+
+public class InputView {
+
+    public static int readPurchaseAmount() {
+        OutputView.printLottoPurchaseAmountPrompt();
+        String input = Console.readLine();
+        ValidationUtils.validateIsNumeric(input);
+        return Integer.parseInt(input);
+    }
+
+    public static List<Integer> readWinningNumbers() {
+        OutputView.printWinningNumbersPrompt();
+        String input = Console.readLine();
+        validateNumbersInput(input);
+        List<Integer> winningNumbers = Arrays.stream(input.split(DELIMITER))
+            .map(Integer::parseInt)
+            .toList();
+        validateWinningNumbers(winningNumbers);
+        return winningNumbers;
+    }
+
+    public static int readBonusNumber() {
+        OutputView.printBonusNumberPrompt();
+        String input = Console.readLine();
+        validateBonusNumber(input);
+        return Integer.parseInt(input);
+    }
+
+    private static void validateBonusNumber(String input) {
+        ValidationUtils.validateNotNull(input);
+        ValidationUtils.validateIsNumeric(input);
+        ValidationUtils.validateNumberInRange(Integer.parseInt(input), LOTTO_START_INCLUSIVE,
+            LOTTO_END_INCLUSIVE);
+    }
+
+    private static void validateWinningNumbers(List<Integer> winningNumbers) {
+        ValidationUtils.validateNumbersSize(winningNumbers);
+        ValidationUtils.validateNoDuplicatedNumberInList(winningNumbers);
+        ValidationUtils.validateNumbersInRange(winningNumbers, LOTTO_START_INCLUSIVE,
+            LOTTO_END_INCLUSIVE);
+    }
+
+    private static void validateNumbersInput(String input) {
+        ValidationUtils.validateNotNull(input);
+        validateProperInput(input);
+        validateNotStartOrEndWithComma(input);
+        validateNoConsecutiveCommas(input);
+        validateNoCommasWithSpaces(input);
+        validateNoSpecialCharactersExceptComma(input);
```

Comment:
개인적인 의견이지만 Comma와 관련된 검증은 하나의 메서드로 분리하는 것이 깔끔할 것 같아요.
```suggestion
        validateInvalidComma(input)
```